docs/_pages/releases.md

```diff
@@ -33,7 +33,7 @@ sidebar:
 * Improved the failure message for `[Not]HaveExplicitProperty` when wrapped in an `AssertionScope` and not implementing the interface - [#2403](https://github.com/fluentassertions/fluentassertions/pull/2403)
 * Improved the failure message for `[Not]HaveExplicitMethod` when wrapped in an `AssertionScope` and not implementing the interface - [#2403](https://github.com/fluentassertions/fluentassertions/pull/2403)
 * Changed `BeEquivalentTo` to exclude `private protected` members from the comparison - [#2417](https://github.com/fluentassertions/fluentassertions/pull/2417)
-
+* When formatting messages, ignoring ArraySegment<> values (they crash the message generation issue) - [#2445](https://github.com/fluentassertions/fluentassertions/pull/2445), [#2511](https://github.com/fluentassertions/fluentassertions/pull/2511)
```

Comment:
I don't know what this sentence means?
This should be on functional level. 
E.g. "Fixed comparing empty `ArraySegment`s for equivalency" or "Fixed using `BeEquivalentTo` on an empty `ArraySegment`".

Reply:
I just cherry-picked all commits, and changed nothing :D